### DISCUSSION TITLE
Song rate & pitch in results screen with wind-up/wind-down doesn't match final rate until result card is clicked

### DISCUSSION THREAD

**Discussion by joshualomelin** [ppy/osu#17262] [Q&A] [answered]
Sorry for the wordy title.

If a beatmap's song continues to play in the results screen while using "wind up" or "wind down," whether because the song has an outro or the map is unfinished, then the rate (and pitch if enabled) of the song will not match the final rate from wind up and wind down. The song's rate will only be corrected after clicking the results card to view the expanded breakdown on the right. However, mods like double time and nightcore will continue to apply the correct rate/pitch without having to click the card.

I am on version 2022.314.0-lazer

Logs:
[database.log](https://github.com/ppy/osu/files/8256390/database.log)
[network.log](https://github.com/ppy/osu/files/8256391/network.log)
[performance.log](https://github.com/ppy/osu/files/8256392/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8256393/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8256394/updater.log)

**Comment by bdach** [comment]
I am not sure how to reproduce this issue, on a quick check behaviour looks correct in the case I checked. Can you show this on a video?

**Reply by joshualomelin** [reply]
Sure, here's a video of it happening:
https://youtu.be/fJ23fpvvDYs

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #17264
